Allow catching connection errors in `catch` node.

reitermarkus opened the pull request:
According to https://discourse.nodered.org/t/node-error-not-caught-in-catch-node/1301/3, a second argument is needed for errors to be catchable.

reitermarkus committed:
Allow catching connection errors in `catch` node.

MPK44 commented:
Hi reitermarkus,
thats a good point. But wouldn't it be better to send the original message instead of the empty string? So a user could identify the command that failed.

reitermarkus commented:
I'm not sure exactly how it works. I though the first message would still be displayed?

reitermarkus committed:
Pass original error as second argument.

MPK44 commented:
I ment `node.error(error, msg)` then after catching it, you can read from `msg.payload` what command caused the problem.

`node.error(error, '')` does not wotk for me.

MPK44 committed:
catchable error with original node message

uses original node message in error node so catch node can identify command that was causing the error
